package/Dockerfile

```diff
@@ -82,31 +103,34 @@ RUN zypper -n addrepo --refresh https://download.opensuse.org/repositories/syste
     zypper --gpg-auto-import-keys ref
 
 RUN zypper -n install nfs-client nfs4-acl-tools cifs-utils sg3_utils \
-    iputils iproute2 qemu-tools e2fsprogs xfsprogs util-linux-systemd python3-pyelftools libcmocka-devel device-mapper && \
+    iproute2 qemu-tools e2fsprogs xfsprogs util-linux-systemd python3-pyelftools libcmocka-devel device-mapper && \
     rm -rf /var/cache/zypp/*
 
 # Install SPDK dependencies
 COPY --from=builder /usr/src/spdk/scripts /spdk/
-RUN if [ "${ARCH}" == "amd64" ]; then \
-        sh /spdk/pkgdep.sh; \
-        rm -rf /spdk; \
-    fi
+RUN sh /spdk/pkgdep.sh && rm -rf /spdk
 
 # Copy pre-built binaries from builder
 COPY --from=builder \
     /usr/local/bin/grpc_health_probe \
     /usr/local/bin/spdk_* \
+    /usr/local/bin/go-spdk-helper \
+    /usr/local/bin/
```

Comment:
Doesn't `COPY /usr/local/bin/` imply copying all files within the directory, including the `grpc_health_probe`, `spdk_*` and `go-spdk-helper`? Same question for `/usr/local/sbin/` and `/usr/sbin/`.

Reply:
The last line `/usr/local/bin/` is the destination directory.

Reply:
I see, it's clear.